src/Actions/Fold.ts

```diff
@@ -0,0 +1,21 @@
+import {commands} from 'vscode';
+import {ActionSelection} from './Selection';
```

Comment:
This is not used and will be marked as error after we update TypeScript to 2.0.3. See #138 .